Pearson-Advance/openedx-themes
# PADV - 541: Remove styles and unnecessary conditional for AWS theme (Ecommerce)

## Ticket

https://agile-jira.pearson.com/browse/PADV-541

## Description

This PR removes styles for aws theme in Ecommerce site. It also removes unnecessary conditional that it shows course price.

## Testing:

- Enable comprehensive theming in Ecommerce.
- Set `aws` on a site.
- Compile the aws theme (`./manage.py update_assets`).
- Verify that the aws theme is applied correctly on Olive.
- Create a course in Studio.
- Add the course previously created in Ecommerce site.
- Add the course to the Ecommerce Basket (_<BASE_URL>/basket/add/?sku=SKU_).

## Screenshots

Courses page:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/4b726964-1f61-4016-aea5-a4686c02b3bd)




Basket page:

- Enrollment type coupon

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/d03e4304-2ffe-4f3e-bec8-f8592f169a83)





- Discount type coupon

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/f900e228-30fa-4ef4-bfb8-630df228d4e9)



Receipt page:

![image](https://github.com/Pearson-Advance/openedx-themes/assets/73655023/2de6345a-6186-4961-b056-6d1b0411ddaa)




## Reviewers
